vpd_tool_impl.hpp

```diff
@@ -190,6 +232,16 @@ class VpdTool
      */
     void readKwFromHw();
 
+    /**
+     * @brief Fix System VPD keyword.
+     * This API provides an interactive way to fix system VPD keywords that are
+     * part of restorable record-keyword pairs. The user can use this option to
+     * restore the restorable keywords in cache or in hardware or in both cache
+     * and hardware.
+     * @return returncode (success/failure).
+     */
+    int fixSystemVPD();
```

Comment:
Why int is required to return success/failure?

Reply:
These are from 1030 and are in 1030,1050